README.md

```diff
@@ -34,6 +34,12 @@ curl -fLo ~/.vim/autoload/plug.vim --create-dirs \
     https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
 ```
 
+If you're using neovim:
```

Comment:
Please make it like the surrounding sections, so this line should be like below. Followed by a space.

`###### Neovim`